docs/index.rst

```diff
@@ -19,6 +19,7 @@ TAPA Documentation
   :caption: Tutorial
 
   tutorial
+  autobridge
```

Comment:
Let's put tutorials under their own folder

Reply:
I propose that we reorganize the contents and have 4 major folders: tutorial, speed (coroutine, xosim, rapidstream in the future), frequency (autobridge), and expressiveness (async_mmap, stream peek, etc.), corresponding to our claim of contribution?